docs/source/API/containers/ScatterView.rst

```diff
@@ -6,16 +6,34 @@
 
 Header File: ``<Kokkos_ScatterView.hpp>``
 
-.. _parallelReduce: ../core/parallel-dispatch/parallel_reduce.html
+Description
+-----------
+`Kokkos View-based <../core/view/view.html>`_ data structure that abstracts over "scatter - contribute" algorithms, where many contributors (over multiple indices) are reduced to fewer output resultants (*e.g.*, sum, product, maximum or minimum). ``ScatterView`` transparently switches between **Atomic**-,  and **Data Replication**-based scatter algorithms.  Typically, a ``ScatterView`` wraps an existing View. 
 
-.. |parallelReduce| replace:: :cpp:func:`parallel_reduce`
 
-Description
+Interface
+-----------
+
+.. cppkokkos:class:: template <class DataType, class LayoutType, class ExecutionSpace, class Operation, class Duplication, class Contribution> ScatterView
+
+
+Parameters
 -----------
 
-.. cppkokkos:class:: template <typename DataType, int Op, typename ExecSpace, typename Layout, int contribution> ScatterView
+*  ``DataType``:  See `Kokkos View DataType <../core/view/view.html>`_
 
-    .. rubric:: Public Member Variables
+*  ``Layout``:  See  `Kokkos View LayoutType <../core/view/view.html>`_
+
+*  ``ExecutionSpace``:  Where the code will be executed (CPU or GPU); typical values are ``Kokkos::DefaultHostExecutionSpace``, ``Kokkos::DefaultExecutionSpace``
```

Comment:
Instead of listing typical values, I would just mention that this defaults to `Kokkos::DefaultExecutionSpace` if not specified.